src/test/java/org/folio/controller/TlrSettingsPublishCoordinatorTest.java

```diff
@@ -0,0 +1,103 @@
+package org.folio.controller;
```

Comment:
File name is `TlrSettingsPublishCoordinatorTest`, but what you are testing is apparently `TlrSettingsController`

Reply:
This is an API test that verifies mod-tlr correctly sends a request to the publish API
```
    wireMockServer.verify(1, postRequestedFor(urlMatching(String.format(PUBLICATIONS_URL_PATTERN, CONSORTIUM_ID)))
      .withRequestBody(equalToJson("""
            {
               "url": "/circulation/settings",
               "method": "POST",
               "tenants": ["college", "university"],
               "payload": {"name":"ecsTlrFeature","value":{"enabled":true}}
            }
        """)));
  }
```